examples/parallel-tempering-2dpmf/parallel-tempering-2dpmf.py

```diff
@@ -281,8 +281,8 @@ def read_file(filename):
         f"bin {i:5d} ({centers_nonzero[i][0]:6.1f}, {centers_nonzero[i][1]:6.1f}) {count_nonzero[i]:12d} conformations"
     )
 
-Ntot = 0  # TODO: this line was one line below??? is this ok?
 x_n = np.zeros([Ntot, 2])  # the configurations
+Ntot = 0  # TODO: this line shouldn't be here? Ntot is not defind previously? is it bin_nonzero?
```

Comment:
I don't know what `Ntot` is supposed to be here, @mrshirts - it definitely can't be zero because otherwise the dimensions for `x_n` are not enough.

Reply:
OK, cleaned up!